### ISSUE TITLE
Problem in exporting color model

### ISSUE THREAD

**Issue by ReallyRad** [nervoussystem/OBJExport#6]
There seems to be an issue in the way faces are indexed for texturing when exporting on Windows 8 with Processing 2.1.1. 
I realized this bug is present in the color cube example. If you import the colored cube 3D model, you will see the order of the colors in the running sketch doesn't correspond to the exported model. 
I am going to check if this happens on Ubuntu as well.
Attached, the problem I am facing at the moment. You can see color is grey on one face where it should be pink. A random face somewhere else in the mesh (in the back) is colored pink instead.
![colored11 error](https://cloud.githubusercontent.com/assets/996367/2767420/f34b096c-ca37-11e3-917d-b78507f703e2.png)

**Comment by ReallyRad** [comment]
So there seems to be an issue with Windows. 
For comparison, here are the textures generated in ubuntu (first) and windows (second)
![cube](https://cloud.githubusercontent.com/assets/996367/2767512/e6cad07c-ca38-11e3-84db-7543603e9a0f.png)
![cube](https://cloud.githubusercontent.com/assets/996367/2767514/eb771c20-ca38-11e3-89f3-6adaaa12f264.png)

**Comment by ReallyRad** [comment]
But the generated model has the same error :  no magenta
![cuberror](https://cloud.githubusercontent.com/assets/996367/2767658/4151d6a2-ca3a-11e3-97ac-ce6e640ddcff.png)
 nor yellow faces, replaced by green and red faces

**Comment by nervoussystem** [comment]
Strange. Are both versions 64-bit? There is a quirk in the way the exporter works in that vertices that are in the same location will be merged into one vertex. That merged vertex can only have one color. This could be related to the error. I could add a check (if it isn't there already) to only merge vertices that have the same color as well as position.

**Comment by ReallyRad** [comment]
Yes, both 64-bits. Will try and investigate a bit further, but I have a deadline to send the files to a 3D printing shop so I may need to go without colors :(
